scripts/generate-rss.php

```diff
@@ -65,7 +71,7 @@
 			<pubDate><?= $published ?></pubDate>
 			<guid><?= $url ?></guid>
 			<? foreach($seSubjects as $seSubject){ ?>
-			<category domain="standardebooks.org"><?= htmlspecialchars($seSubject, ENT_QUOTES, 'UTF-8') ?></category>
+			<category domain="<?= preg_replace('/^[^:]+:\/\//ius', '', $webUrl) ?? '' ?>"><?= htmlspecialchars($seSubject, ENT_QUOTES, 'UTF-8') ?></category>
```

Comment:
I don't think we need to customize this line. The domain of the category is always going to be standardebooks.org because that's the name of the project.